### PR TITLE
Support Android platform

### PR DESCRIPTION
**Pull request by ghsdh3409**
- We replaced index.ios.js with index.js (in package.json).
- In index.js, we used platform-specific code for supporting both iOS and Android.
- We added Android-related codes (default setting for RN Android project) in .gitignore file.
- All the interfaces to native module is designed as same as the existing iOS code (backward-compatible).
- Android modules also supports ‘distance’ field in addition to ‘proximity’ field.

This implementation is pair-programmed with @ohtangza.

### CODE REVIEW

**Comment by ohtangza** [comment]
This is implementation of Issue #2.
We tested it both for iOS 10 (iPhone 6) and Android 6.0.1 (Galaxy S6), and it worked very well :)

**Comment by williambout** [comment]
Hey guys, thanks for the pull request.
I don't have any Android device, but I'll try to get one in order to test this.
Out of curiosity, have tried https://github.com/rhysowen/react-native-proximity-android ?

**Comment by ohtangza** [comment]
@ghsdh3409 Can you tell the details with react-native-proximity-android? 

@williambout We strongly believe that it is better to have iOS and Android in single project if the functionality is something in common for both platforms. The reason that we submitted pull request here other than react-native-proximity-andorid is 1) we are more familiar with Android, 2) we were more favor of your repository because you were more active in Github.

I will plan to update README.md for Android support with further details.

By the way, in Android module, we also supports 'distance' field which is not currently supported in iOS module. Isn't it supported in iOS side?

**Comment by ohtangza** [comment]
I just pushed updated README.md on this branch. 

@williambout After merging it, could you publish the higher version on npm so that we can add it as our project' dependency. We plan to use this library for our app which will be on production soon :)

File: README.md [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
 Import the library
 
 ```javascript
 import Proximity from 'react-native-proximity';
 ```
 
-###addListener(callback)
-
+### addListener(callback)
+The callback function returns object with *proximity* and *distnace* property. If *proximity* is true, it means the device is close to an object. *distance* is only supported in Android.
```
Comment (williambout) [inline README.md]: There is a small typo here.

**Comment by williambout** [comment]
@ohtangza @ghsdh3409  I just tried on Android and iOS and had no issue 👍. I'll merge this a soon as the typo in the readme is fixed.
I'll release this as v2.0.0 on npm.

**Comment by ohtangza** [comment]
@williambout Sorry for my little late response. I just corrected it with new commit. 👍

**Pull request merged**